lib/Graph/Graph.cpp

```diff
@@ -1942,6 +1942,25 @@ SpaceToDepthNode *Function::createSpaceToDepth(llvm::StringRef name,
   return addNode(new SpaceToDepthNode(name, outTy, input, blockSize));
 }
 
+ResizeNearestNode *Function::createResizeNearest(llvm::StringRef name,
+                                                 NodeValue input,
+                                                 float heightScale,
+                                                 float widthScale) {
+
+  auto inputDim = input.dims();
+  assert(inputDim.size() == 4 && "Dimension size of 4 is expected.");
```

Comment:
we've started migration to DCHECK from glog for better error messages. Could you modify and also add actual size in case there is a mismatch.